view/frontend/web/js/view/payment/method-renderer/adyen-paypal-method.js

```diff
@@ -35,6 +35,12 @@ define(
 
                 this.actionComponent = component.handleAction(action);
             },
+            handleOnFailure: function(response, component) {
+                const self = this;
+                self.isPlaceOrderAllowed(true);
```

Comment:
Since execution happening in the same context, you don't need to define `self` and use it instead of `this`. Feel free to use `this` directly here.